src/overmap.cpp

```diff
@@ -3713,13 +3660,14 @@ const unsigned &oter_id::operator=(const int &i)
     return _val;
 }
 // ter(...) = "rock"
-oter_id::operator std::string() const
+namespace { std::string const debug_dummy_string; }
```

Comment:
It could be, but since c++11, function local statics have had their initialization and access rules updated to play well in cases where race conditions can exist. This causes some extra overhead, and since this was ostensibly a tweak made for performance (and I don't see CATA going multithreaded anytime soon...)

[Assembly listing](http://pastebin.com/MWfLUEe3)

On the other hand, the impact is probably pretty minor and I could move this inside the function.